code/modules/mob/living/carbon/xenomorph/abilities.dm

```diff
@@ -1322,6 +1322,8 @@
 		return
 	if(!iscarbon(A))
 		return FALSE
+	if(owner.status_flags & INCORPOREAL)
+		return FALSE
```

Comment:
this can be before parent to reduce overhead

Reply:
huh this is psydrain thats not related to me whaa